lib/JMAP/Tester.pm

```diff
@@ -216,7 +220,11 @@ sub request {
 
   my %default_args = %{ $self->default_arguments };
 
-  for my $call (@$calls) {
+  my $request = _ARRAY0($input_request)
+              ? { methodCalls => $input_request }
+              : { %$input_request };
```

Comment:
Why not just use `$input_request` here?

(update: I guess you want to keep the original around to inspect later. Never mind, then!)